pkgs/by-name/va/vagrant/package.nix

```diff
@@ -118,6 +118,6 @@ in buildRubyGem rec {
     license = licenses.bsl11;
     maintainers = with maintainers; [ tylerjl ];
     platforms = with platforms; linux ++ darwin;
-    broken = true; # build fails on darwin and linux
+    broken = stdenv.hostPlatform.isDarwin;
```

Comment:
I tested that this builds and runs on Darwin, so we can remove this.

Reply:
Thank you! I'll remove the `broken` stanza.